packages/vscode-boxel-tools/src/file-system-provider.ts

```diff
@@ -359,15 +360,12 @@ export class RealmFS implements vscode.FileSystemProvider {
 
     try {
       let headers = {
-        Accept: 'application/vnd.api+json',
+        Accept: SupportedMimeType.JSONAPI,
```

Comment:
I don't understand why we set this here--shouldn't we always be using card-source header regardless--just like the monaco editor does in the host app?

Reply:
I don’t know why it was like this, I was just changing what worked to the extracted variable. But I changed to `CardSource` and it worked, thanks.